### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by tamoorsaeed22**
### Implementations in this project

- Add  JSX elements to the project 
- Final design matches zeplin's design 
- Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/book.js [hunk 1]
```diff
@@ -1,12 +1,37 @@
...
+          <button type="button" className={style.btn}>Edit</button>
+        </div>
+      </div>
+      <ProgressBar />
+      <div style={line} />
+      <div className={style.chapter}>
+        <div>
+          <p className={style.chapter_heading}>CURRENT CHAPTER</p>
+          <p className={style.chapter}>{`Chapter ${chapter()}`}</p>
+        </div>
```
Comment (mikethreels) [inline src/components/book.js:34]: - [x] The progress bar of all books is not aligned as you can see below, due to the titles being a different length for each book, you can fix this by giving the book title container a relative width.
afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Tamoor, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job adding the styling

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/book.js (hunk 1 above)
Comment (tamoorsaeed22) [reply src/components/book.js:34]: @mikethreels hey mike the titles are a different length for each book, I have fixed the issue so they are aligned now .
as you can see below 
Screenshot (18)

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @tamoorsaeed22 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


 ## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**